Cloudbuild config for a nightly e2e system test

dinagraves opened the pull request:
Adding an e2e test config for nightly tests.  Runs the setup script, checks the tables for data, and then deletes the resources and turns off the API's.

donmccasland commented on e2e_tests.cloudbuild.yaml:
```diff
@@ -0,0 +1,84 @@
+        pip3 install --upgrade pip
+        cd setup/
+        export FOURKEYS_PROJECT=${PROJECT_ID}
+        printf "n\ny\n2\n1\nn\ny\n" | ./setup.sh
```
This works, but is less readable.  Potentially we should save this kind of information from a run of setup.sh and import it.

donmccasland replied:
Also, it would be great to do some comment explanations here.

dinagraves replied:
Added comments -- is it clearer?

dinagraves commented:
Added additional changes to help with flakiness:  

- Addressed the issue of existing secret #40 
- Changed the scheduler to manually run the transfer #20

donmccasland commented on e2e_tests.cloudbuild.yaml:
```diff
@@ -0,0 +1,97 @@
+        gcloud services disable bigquerydatatransfer.googleapis.com 
+        gcloud services disable bigqueryconnection.googleapis.com 
+        gcloud services disable secretmanager.googleapis.com 
+        set +x
```
Looks like it needs newline at the end here.
